mapper/orm/src/test/java/org/hibernate/search/mapper/orm/model/impl/HibernateOrmBootstrapIntrospectorAccessTypeTest.java

```diff
@@ -26,117 +26,138 @@
 import jakarta.persistence.Transient;
 
 import org.hibernate.search.mapper.pojo.model.spi.PojoRawTypeModel;
+import org.hibernate.search.util.common.reflect.spi.ValueHandleFactory;
 import org.hibernate.search.util.common.reflect.spi.ValueReadHandle;
 
-import org.junit.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
-public class HibernateOrmBootstrapIntrospectorAccessTypeTest
+class HibernateOrmBootstrapIntrospectorAccessTypeTest
 		extends AbstractHibernateOrmBootstrapIntrospectorPerReflectionStrategyTest {
 
-	@Test
-	public void entity_defaultFieldAccess() {
-		HibernateOrmBootstrapIntrospector introspector = createIntrospector( EntityWithDefaultFieldAccess.class );
+	@ParameterizedTest(name = "Reflection strategy = {0}")
+	@MethodSource("params")
```

Comment:
yeah 😔,
before this, we were using something similar but on a smaller scale:
```java
@Parameterized.Parameters(name = "{0} - {1}")
Obecjt[][] params() {
```
I suppose I can remove the names 😃... on the other hand, with that in place, the test names should better match with what we had, if we'd need to compare the results...
Also in some cases, parameters are lambdas so that wouldn't be a much useful output in the tests too...
soooooooo IDK 🤷🏻‍♂️

Reply:
> on the other hand, with that in place, the test names should better match with what we had, if we'd need to compare the results...

Hmm ok, we can do this change last then. Or in a separate commit. Or not at all, we'll see.

>  Also in some cases, parameters are lambdas so that wouldn't be a much useful output in the tests too...

Not useful, but hopefully won't hurt either.